CMakeModules/build_forge.cmake

```diff
@@ -42,13 +42,13 @@ ELSE()
     SET(byproducts BYPRODUCTS ${forge_location})
 ENDIF()
 
-SET(FORGE_VERSION 0.9.0)
+SET(FORGE_VERSION 0.9.1)
 
 # FIXME Tag forge correctly during release
 ExternalProject_Add(
     forge-ext
     GIT_REPOSITORY https://github.com/arrayfire/forge.git
-    GIT_TAG v${FORGE_VERSION}
+    GIT_TAG devel
```

Comment:
@shehzan10 I have merged forge PR, please change it back to `${FORGE_VERSION}`. I wanted to talk about this for a while now. Having devel here is not good because if any breaking change of forge has been pushed to devel in the mean while. It cause build failures in arrayfire until related changes are incorporated into arrayfire. We should either keep it as tag or complete commit number.